index.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, user-scalable=no"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <div class="container">
+          <div class="headerInner">
+            <div class="logo">
+              <img src="image/skoutTop.png" alt="Skout" class="skoutTop"/>
+              <img src="image/skoutGrey.png" alt="Skout" class="skoutGrey none"/>
+            </div>
+
+            <div class="containerHeaderMenu none">
+              <div class="wrapperHeaderMenu">
+                <nav class="headerMenu">
+                  <a href="#" class="currentLink">WHY SKOUT</a>
+                  <a href="#">PRODUCTS</a>
+                  <a href="#">PARTNERS</a>
+                  <a href="#">COMPANY</a>
+                  <a href="#">TIPS&TOOLS</a>
+                  <a href="#">BLOG</a>
+                </nav>
+              </div>
+              <div class="enterButton">
+                <button>GET SKOUT</button>
+                <button>LOG IN</button>
+              </div>
+            </div>
+
+            <nav class="gamburgerContainer">
+              <button class="gamburger gamburgerIcon"></button>
+            </nav>
+          </div>
+
+          <div class="headerText"><span>Tips&Tools</span></div>
+        </div>
+      </header>
+
+      <nav class="tabs">
+        <div class="lineTabs"></div>
```

Comment:
this line can be made by css in tabs element

Reply:
done